src/main/webapp/jsp/instructorFeedbackResultsTop.jsp

```diff
@@ -10,12 +10,16 @@
 <r:sessionPanel sessionPanel="${data.sessionPanel}" />
 
 <c:if test="${not (noResponses and data.allSectionsSelected and showAll)}">
-  <r:filterPanel filterPanel="${data.filterPanel}"
-      showAll="${showAll}" />
+  <r:filterPanel filterPanel="${data.filterPanel}"/>
 </c:if>
 <br>
+<div style="margin-top:10px">
```

Comment:
I feel that wrapping the status message in an additional `div` just to add a top margin in this particular page only (`instructorFeedbackResultsTop.jsp`) makes it inconsistent with the other usages of `statusMessage.tag` in the other pages. I prefer the original approach of adding a bottom margin to the filter panel.